decidim-core/app/views/pages/decidim_page.html.erb

```diff
@@ -10,7 +10,7 @@
   <div class="row">
     <div class="columns large-8">
       <div class="static__content">
-        <%== translated_attribute page.content %>
+        <%= sanitize translated_attribute page.content %>
```

Comment:
I wouldn't sanitize here to give more freedom to the user, as they might want to include iframes or such.